templates/review-form.html

```diff
@@ -18,7 +18,11 @@ <h2 class="text-center">Leave a Review</h2>
                 {{ field(class='form-control') }}
                 {% endif %}
             {% endfor %}
-            <button class="btn btn-primary my-2">Save changes!</button>
+            <div id="uploadProgressContainer" style="display: none;">
```

Comment:
be consistent in class and id names. Either snake-case or camelCase

Reply:
done